native-gradle-plugin/src/main/java/org/graalvm/buildtools/gradle/NativeImagePlugin.java

```diff
@@ -332,26 +345,46 @@ private void configureAutomaticTaskCreation(Project project,
                 task.setDescription("Executes the " + options.getName() + " native binary");
                 task.getImage().convention(imageBuilder.flatMap(BuildNativeImageTask::getOutputFile));
                 task.getRuntimeArgs().convention(options.getRuntimeArgs());
+                var useLayers = options.getLayers()
+                    .stream()
+                    .filter(layer -> layer instanceof UseLayerOptions)
+                    .collect(Collectors.toList());
```

Comment:
Since we updated the minimal project version to `17`, can't we use just `toList()` here?